DfciPkg/Docs/Dfci_Feature.md

```diff
@@ -79,14 +79,10 @@ included in the UEFI BIOS image.
 The DFCI source code and public certificate are available on GitHub under a permissive open
 source license (SPDX-License-Identifier: BSD-2-Clause-Patent).
 
-Source code hard links to the source
-* <https://github.com/microsoft/mu_plus/search?q=DfciPkg>

```

Comment:
STATIC DFCI_SETTING_ID_STRING mAllCameraSettings[] = {
    DEVICE_SETTING_ID__FRONT_CAMERA,
    DEVICE_SETTING_ID__REAR_CAMERA,
    DEVICE_SETTING_ID__IR_CAMERA,
    NULL
};
STATIC DFCI_SETTING_ID_STRING mAllCpuAndIoVirtSettings[] = {
    DEVICE_SETTING_ID__ENABLE_VIRT_SETTINGS,
    NULL
};

STATIC DFCI_GROUP_ENTRY mMyGroups[] = {
    { DFCI_SETTING_ID__ALL_CAMERAS,     (DFCI_SETTING_ID_STRING *) &mAllCameraSettings },
    { DFCI_SETTING_ID__ALL_AUDIO,       (DFCI_SETTING_ID_STRING *) &mAllAudioSettings },
    { DFCI_SETTING_ID__ALL_RADIOS,      (DFCI_SETTING_ID_STRING *) &mAllRadiosSettings },
    { DFCI_SETTING_ID__EXTERNAL_MEDIA,  (DFCI_SETTING_ID_STRING *) &mExternalMediaSettings },
    { DFCI_SETTING_ID__ENABLE_NETWORK,  (DFCI_SETTING_ID_STRING *) &mOnboardNetworkSettings },
    { DFCI_SETTING_ID__ALL_CPU_IO_VIRT, (DFCI_SETTING_ID_STRING *) &mAllCpuAndIoVirtSettings },
    { NULL,                             NULL }
};

/**
 * Return a pointer to the Group Array to DFCI
 *
 */
DFCI_GROUP_ENTRY *
EFIAPI
DfciGetGroupEntries (VOID) {

    return (DFCI_GROUP_ENTRY *) &mMyGroups;
}